gui/mozregui/main.py

```diff
@@ -97,11 +98,33 @@ def show_about(self):
     def edit_global_prefs(self):
         change_prefs_dialog(self)
 
+    @Slot(str)
+    def on_log_received(self, log_message):
+        self.ui.log_information_view.appendPlainText(log_message)
+        counter = self.ui.log_information_view.blockCount()
+        print "counter is ", counter
+        if(counter > 1000):
+            self.ui.log_information_view.clear()
```

Comment:
hmm this will erase everything. I think that we should try to just remove the first block in this case, and after append the log line. Maybe this can help as a start: http://stackoverflow.com/questions/10417795/remove-a-line-block-from-qtextedit